test/test_autoextract.py

```diff
@@ -105,3 +106,15 @@ def test_timeout():
     out = mw.process_request(req, spider)
     assert out is not None
     assert out.meta['download_timeout'] == 10000
+
+
+def test_meta_extra():
+    config = dict(MW_SETTINGS)
+    mw = _mock_mw(spider, config)
+    meta = dict(AUTOX_META)
+    meta['autoextract']['extra'] = {'some': 'stuff'}
```

Comment:
you're modifying AUTOX_META inplace here, because dict(AUTOX_META) is a shallow copy

Reply:
Good point :)
I would have noticed in the next text, when failing for weird reasons
Fixing now